apps/vmq_passwd/c_src/Makefile

```
@@ -3,7 +3,7 @@ CFLAGS_EXTRA =
 
 ifeq ($(shell uname -s), Darwin)
 	# OSX with homebrew
-	OPENSSL_DIR ?= /usr/local/opt/openssl
+	OPENSSL_DIR ?= $(brew --prefix openssl)
```

I thought a little bit more and tested with self compiled open SSL instead of brew, would be good to keep the default install location in case the user uses self compiled homebrew or macports. Something like this should work:

```suggestion
    HAS_BREW := $(shell command -v brew;)

	ifdef HAS_BREW
		OPENSSL_DIR ?= $(shell brew --prefix openssl)
	endif

	# Default dir
	OPENSSL_DIR ?= /usr/local/opt/openssl

```

---

hmm nice idea, i am using your changes